lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -0,0 +1,21 @@
+<div class="exam-timer">
+    <%- gettext("You are taking") %>
+    <a href='<%= exam_url_path %>' >
+        <%= exam_display_name %>
```

Comment:
Make sure to escape the display name to avoid XSS (test this for XSS using a script tags).